### ISSUE TITLE
[Bug]: load tpcc history table data was hung on eks with 2*cn  

### ISSUE THREAD

**Issue by aressu1985** [matrixorigin/matrixone#9181]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):nightly-9ca72b6
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

when loading tpcc table bmsql_history data on eks with 2*cn , it can not be succeed. 

<img width="1322" alt="image" src="https://user-images.githubusercontent.com/47846308/234500792-0a9de03a-1bb6-47e7-b07b-639b15c8a075.png">

the error log:
[load-hung.tar.gz](https://github.com/matrixorigin/matrixone/files/11330144/load-hung.tar.gz)

### Expected Behavior

loading data for all tpcc table can be succeed.

### Steps to Reproduce

```Markdown
1. https://github.com/matrixorigin/mo-auto-test.git
2. modify /.github/workflows/regression-on-distributed-mode-no-build.yml, 
   set jobName to tpcc_benchmark_test
   set Image-default to nightly-9ca72b6
3. make a pr and the test can ben run
```

**Comment by w-zr** [comment]
Should be fixed after #9208.

**Comment by aressu1985** [comment]
fixed
